src/Collectors/EventDataCollector.php

```diff
@@ -136,4 +136,10 @@ private function toMilliseconds(float $time): float
     {
         return round($time * 1000, 3);
     }
+
+    public function reset(): void
```

Comment:
Could you add this method to the `AG\ElasticApmLaravel\Contracts\DataCollector` interface?

Reply:
Done. Waiting on the checks to confirm. I rebased and squashed the commits to clean this up as well.